floris/wind_data.py

```diff
@@ -1551,7 +1638,7 @@ def plot(
             rects = []
             freq_table_sub = freq_table[wd_idx, :].flatten()
             for var_idx, ws in reversed(list(enumerate(var_bins))):
-                plot_val = freq_table_sub[:var_idx + 1].sum()
+                plot_val = freq_table_sub[: var_idx + 1].sum()
```

Comment:
I presume this is something from ruff-format? It's a strange one, I'm not a fan, but so be it...

Reply:
Exact, we can take it out if you want